internal/events/trigger/retrigger_event.go

```diff
@@ -21,19 +21,24 @@ func RefireEvent(id string, p TriggerParameters) (string, error) {
 
 	p.Transport = res.Transport
 
-	e, err := types.GetByTriggerAndTransport(p.Event, p.Transport)
+	e, err := types.GetByTriggerAndTransport(res.Event, p.Transport)
```

Comment:
This actually fixes the bug you noted; `p.Event` doesn't have the trigger, only the database `res`ponse. 

Will merge this.